src/frontend/packages/lib_tests/src/render.tsx

```diff
@@ -3,14 +3,14 @@ import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
 import {
   RenderResult,
   RenderOptions as TestingLibraryRenderOptions,
-  act,
   screen,
   render as testingLibraryRender,
 } from '@testing-library/react';
 import { Grommet, ResponsiveContext, ThemeType } from 'grommet';
 import MatchMediaMock from 'jest-matchmedia-mock';
 import { BreadCrumbsProvider, GlobalStyles, Nullable, theme } from 'lib-common';
 import React, { CSSProperties, ComponentProps, ReactElement } from 'react';
+import { act } from 'react-dom/test-utils';
```

Comment:
I remember this method is deprecated in favor of `react/test-utils`.
https://react.dev/warnings/react-dom-test-utils

But I don't understand why you cannot continue to use `act` from `testing-library/react` ?